cadquery/assembly.py

```diff
@@ -488,11 +490,22 @@ def save(
         elif exportType == "VRML":
             exportVRML(self, path, tolerance, angularTolerance)
         elif exportType == "GLTF":
-            exportGLTF(self, path, True, tolerance, angularTolerance)
+            # Handle the binary option for GLTF export
```

Comment:
I've opened #1420.  IMO, `save` must also recognize the .glb extension automatically.